exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/ProjectPrel.java

```diff
@@ -81,6 +83,18 @@ public SelectionVectorMode getEncoding() {
 
   @Override
   public boolean needsFinalColumnReordering() {
+    for (RexNode expr : this.exps) {
+      if (expr.getKind() == SqlKind.OTHER_FUNCTION &&
+          expr instanceof RexCall &&
+          ((RexCall) expr).getOperator().getName().equalsIgnoreCase("CONVERT_FROMJSON")) {
```

Comment:
Can you pls add a comment that convert_fromjson() currently can only used as a top level function (it can not be nested in other function)? Otherwise, we have to walk through the expression tree to check the existence of this function. 

Tried with convert_to(convert_from(t.jsonstr, 'json'), 'json'), and it hit NPE in execution.